cli/src/helper.rs

```diff
@@ -61,6 +62,53 @@ pub async fn get_alias(prompt: &str) -> Result<String, Error> {
     }
 }
 
+pub async fn get_address(prompt: &str) -> Result<Bech32Address, Error> {
+    loop {
+        let input = Input::<String>::new().with_prompt(prompt).interact_text()?;
+        if input.is_empty() || !input.is_ascii() {
+            println_log_error!("Invalid input, please choose a non-empty address consisting of ASCII characters.");
+        } else {
+            return Ok(Bech32Address::from_str(&input)?);
+        }
+    }
+}
+
+pub async fn get_bip_path(prompt: &str) -> Result<Bip44, Error> {
+    loop {
+        let input = Input::<String>::new().with_prompt(prompt).interact_text()?;
+        if input.is_empty() || !input.is_ascii() {
+            println_log_error!("Invalid input, please choose a non-empty address consisting of ASCII characters.");
```

Comment:
Why address?

Reply:
well, you know ... there's this incredibly efficient but error-prone coding style ... what was is called? uhm ... co ... cop ... copy & paste or somethin :trollface:

Reply:
Thanks, fixed